react/features/large-video/subscriber.js

```diff
@@ -0,0 +1,15 @@
+// @flow
+
+import { StateListenerRegistry } from '../base/redux';
+import VideoLayout from '../../../modules/UI/videolayout/VideoLayout';
+
+/**
+ * For auto-pin mode, listen for changes to the known media tracks and look
```

Comment:
Probably want to change doc this but it still kind of makes sense.

Reply:
Yep. Copy/paste error 🤦‍♂ 

It is fixed now!